javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,82 @@
+console.log("Hello"); //Initial code
+
+//Age-ify (A future age calculator)
+const yearOfBirth = 1989;
+const yearFuture = 2050;
+
+//with using - arithmatic operator we can calculate the age.
+let myAge  = yearFuture  - yearOfBirth;
+
+console.log("I will be " + myAge + " years old in " + yearFuture);
+
+
+//Goodboy-Oldboy (A dog age calculator)
+ const dogYearOfBirth = 2017;
+ const dogYearFuture = 2027;
+ const dogYear = dogYearFuture - dogYearOfBirth;
+ const dogHumunsYear = 7 * dogYear ;
+ let shouldShowResultInDogYears = "true";
+
+
+ if (shouldShowResultInDogYears == "true") {
+     console.log("My dog will be " + dogYear+ " dog years old in " + dogYearFuture )
+ } else {
+     console.log("My dog will be " + dogHumunsYear+ " humuns years old in " + dogYearFuture )
+ };
+
+ //Housey pricey (A house price estimator)
+
+//For Peter house
```

Comment:
Imagine you can put all values in the same array for both Peter and Julia, for example,
`const volumeInMeters = [8*10*10,5*11*8]` `const gardenSizeInM2 = [100,70]`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?

Reply:
I will work on this and will commit it .